packages/story-editor/src/constants/index.js

```diff
@@ -21,7 +21,7 @@ export * from './performanceTrackingEvents';
 
 export const ADMIN_TOOLBAR_HEIGHT = 32;
 export const HEADER_HEIGHT = 64;
-export const HEADER_GAP = 16;
+export const HEADER_GAP = 10;
```

Comment:
Hm, is this reverting any of the changes requested by design / does this have any visual impact that needs to be followed up?

Reply:
It looks like there are more places that are sensitive to the actual canvas size on the screen (my first fix didn't fix all of them), so I thought it would be better to get the old size back by reducing the whitespace by a tiny bit, I see no cons to that solution and we get the old/bigger canvas (why make it smaller?) and the whitespace around the canvas is still big IMO (we could reduce it even more, but we would have to adjust the tests). WDYT @swissspidy ?

Reply:
Do you perhaps have a before/after screenshot for this change?

Reply:
Added to the table^

Reply:
Thanks! The difference is indeed minimal. Seems fine to me 👍